src/main/resources/templates/add_category.html

```diff
@@ -14,18 +14,33 @@
 <div class="container">
     <h1> Kategorie</h1>
 
-    <ul th:each="category : ${categoryList}">
-        <div>
-            <label th:text="${category.categoryName}"></label>
-            <a method=delete th:href="@{'/deleteCategory/'+${category.categoryName}}"
-               class="btn btn-primary mb-2">
-                Usun Kategorie</a>
-        </div>
-    </ul>
+    <table class="table table-striped table-bordered">
+        <thead class="thead-dark">
+        <tr>
+            <th>Kategoria</th>
+            <th>Akcje</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr th:each="category: ${categoryList}">
+            <td>
+                <label th:text="${category.categoryName}"></label>
+            </td>
+            <td> <a method=delete th:href="@{'/deleteCategory/'+${category.categoryName}}"
+                    class="btn btn-primary mb-2">
+                    Usun Kategorie</a></td>
```

Comment:
change `<meta charset="ISO-8859-1">` to `<meta charset="UTF-8">` and add polish chars.

Reply:
Fixed